src/pages/container/EventsTable/util.tsx

```diff
@@ -0,0 +1,35 @@
+import { Link } from "react-router-dom";
+import { getTaskRoute } from "constants/routes";
+import { PodEventsQuery } from "gql/generated/types";
+import { Unpacked } from "types/utils";
+
+export const getEventCopy = (
+  event: Unpacked<PodEventsQuery["pod"]["events"]["eventLogEntries"]>
+) => {
+  const { eventType, data } = event;
+  const taskLink = (
+    <Link to={getTaskRoute(data.taskID, { execution: data.taskExecution })}>
+      {data.task?.displayName}
+    </Link>
+  );
+  switch (eventType) {
+    case "STATUS_CHANGE":
+      return (
+        <span>
+          Container status changed from {data?.oldStatus} to {data?.newStatus}.
```

Comment:
Can we emphasize the statuses we do so on the host status page.
![image](https://user-images.githubusercontent.com/4605522/230173408-d820950c-95c1-46dd-a62d-7340328abfe6.png)